### DISCUSSION TITLE
Artillery sending traces to otel collector

### DISCUSSION THREAD

**Discussion by danielvan1** [artilleryio/artillery#2752] [Artillery]
Hi,

I am currently trying to send traces from my pods which run an `artillery` script to an `otel` collector which runs in the same Kubernetes cluster on the same node. It is currently not possible for me to do this and I get an error saying:

![Screenshot 2024-05-15 at 16 33 31](https://github.com/artilleryio/artillery/assets/24315292/b8299fa7-bb52-4136-95da-4553781ccd46)

To deploy the otel collector I use the [default helm chart](https://opentelemetry.io/docs/kubernetes/helm/collector/) where I configured it such that we disabled metrics and logging pipelines and non-otlp receivers.

My current plugins in my `artillery-config.yml` looks like this: 

![Screenshot 2024-05-15 at 16 31 31](https://github.com/artilleryio/artillery/assets/24315292/8b317c30-cd29-4c02-b96f-04fe243e9378)

Does anyone have an idea how to fix this?

**Comment by InesNi** [comment]
Hi @danielvan1 👋🏻 ,

Thank you for getting in touch, 

I believe you might just be missing the `http://` in your endpoint making it `http://opentelemetry-collector:4318/v1/traces`

I hope that works, but if it still doesn't could you share some more information if possible:
- Your exact collector configuration if possible
- Artillery and Node.js version you're running
- Are any other services using the same endpoints/pipelines?

**Reply by danielvan1** [reply]
Thanks for the answer. 

Unfortunately, it did not work. Adding the `http://` prefix to my endpoint ended up with another error:
![Screenshot 2024-05-15 at 19 13 41](https://github.com/artilleryio/artillery/assets/24315292/4e3e6474-3dc1-4280-8ce9-4ea0fa949dd0)
Instead, I get an `ENOTFOUND` error which suggests the first URL might be correct and adding `http://` might be an non existing url.


### Answers:

> Your exact collector configuration if possible

I have attached my `values.yml` file [values.yaml.zip](https://github.com/artilleryio/artillery/files/15333993/values.yaml.zip)

> Artillery and Node.js version you're running

I use the latest docker image: `artilleryio/artillery:latest`

> Are any other services using the same endpoints/pipelines

No
